app/experimenter/kinto/diagrams/diagrams.sequence

```diff
@@ -9,22 +9,24 @@ participant Remote Settings UI
 participant Remote Settings Backend
 
 
+### Launching
```

Comment:
A *little* confused about the formatting in this file. Looks like it's got markdown-style headers, but in reverse. Tried pasting it into sequencediagram.org, but it looks like that doesn't quite work as one big document. 

Is there any build script that produces the pngs from this file? Or is it all manual copy & paste of individual diagrams? 

If the latter, that would be useful to add to the README. Could also be worth looking into a build script at some point if we end up modifying this further. [Mermaid has sequence diagrams and build scripts](http://mermaid-js.github.io/mermaid/#/sequenceDiagram) that we've used over on FxA before, FWIW.

Reply:
Oh neat.  Yeah it's just copy paste.  Those are comments, not headers.